lib/appbundler/app.rb

```diff
@@ -90,7 +90,7 @@ def ruby_relative_path
     end
 
     def shebang
-      "#!#{ruby}\n"
+      "#!#{ruby} --disable-gems\n"
```

Comment:
Disables `rubygems` from being included so we have a chance to clear `GEM_HOME` and `GEM_PATH`. We require it ourselves (`require "rubygems"`) below.